src/frontend/screens/Settings/components/EnableMsync.tsx

```diff
@@ -0,0 +1,44 @@
+import ContextProvider from 'frontend/state/ContextProvider'
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import SettingsContext from '../SettingsContext'
+import useSetting from 'frontend/hooks/useSetting'
+import { ToggleSwitch } from 'frontend/components/UI'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
+
+
+const EnableMsync = () => {
+    const { t } = useTranslation()
+    const { platform } = useContext(ContextProvider)
+    const { isMacNative } = useContext(SettingsContext)
+    const isMac = platform === 'darwin'
+    const [enableMsync, setEnableMsync] = useSetting('enableMsync', false)
+
+    if (!isMac || isMacNative){
+        return <></>
+    }
+
+    return(
```

Comment:
Prettier will probably complain about these, you need to fix the styling by running
`yarn prettier-fix`